ai_flow/model/internal/conditions.py

```diff
@@ -36,24 +37,56 @@ def is_met(self, event: Event, context: Context) -> bool:
         return True
 
 
+def is_match_event(event_key: EventKey, event: Event) -> bool:
+    if event_key.namespace is not None and event_key.namespace != event.event_key.namespace:
```

Comment:
If event_key.namespace is None and event.event_key.namespace is not None, the first check statement will pass, but I don't think it's expected